samples/app-auth/nodejs/assets/sample.json

```diff
@@ -1,15 +1,15 @@
 [
   {
-    "name": "officedev-microsoft-teams-samples-app-auth-nodejs",
+    "name": "officedev-microsoft-teams-samples-app-complete-auth-nodejs",
     "source": "officeDev",
-    "title": "Microsoft Teams Authentication Sample",
-    "shortDescription": "This sample demonstrates authentication in Microsoft Teams using bot and tab.",
-    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-auth/nodejs",
+    "title": "Authentication complete sample",
+    "shortDescription": "This sample demos Azure AD and facebook authentication feature in bot,tab and messaging extension.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-complete-auth/nodejs",
     "longDescription": [
-      "This sample demonstrates authentication in Microsoft Teams using bot and tab."
+      "This sample demos Azure AD and facebook authentication feature in bot,tab and messaging extension."
     ],
-    "creationDateTime": "2018-02-08",
-    "updateDateTime": "2022-06-20",
+    "creationDateTime": "2022-02-16",
```

Comment:
Do we need to update created date as well?

Reply:
We can change updated date only